docs/en/sql-reference/data-types/datetime64.md

```diff
@@ -9,33 +9,35 @@ Allows to store an instant in time, that can be expressed as a calendar date and
 
 Tick size (precision): 10<sup>-precision</sup> seconds
 
-Syntax:
+**Syntax:**
 
 ``` sql
 DateTime64(precision, [timezone])
 ```
 
 Internally, stores data as a number of ‘ticks’ since epoch start (1970-01-01 00:00:00 UTC) as Int64. The tick resolution is determined by the precision parameter. Additionally, the `DateTime64` type can store time zone that is the same for the entire column, that affects how the values of the `DateTime64` type values are displayed in text format and how the values specified as strings are parsed (‘2020-01-01 05:00:01.000’). The time zone is not stored in the rows of the table (or in resultset), but is stored in the column metadata. See details in [DateTime](../../sql-reference/data-types/datetime.md).
 
+Supported range from January 1, 1925 till December 31, 2238.
```

Comment:
```
 :) select toDateTime64(0x7FFFFFFFFFFFFFFF, 1)

SELECT toDateTime64(9223372036854775807, 1)

Query id: 67fe1d07-e984-4be5-ab72-91c8a78a3d34

┌─toDateTime64(9223372036854775807, 1)─┐
│                2282-12-31 03:00:00.0 │
└──────────────────────────────────────┘

1 rows in set. Elapsed: 0.002 sec. 

 :) select toDateTime64(-0x8000000000000000, 1)

SELECT toDateTime64(-9223372036854775808, 1)

Query id: e161ce59-ea6c-4643-884c-bbf43de5cff3

┌─toDateTime64(-9223372036854775808, 1)─┐
│                 1925-01-01 02:00:00.0 │
└───────────────────────────────────────┘
```

Reply:
@KochetovNicolai That means, the description in the #9404 is not correct?

Reply:
Well, It looks like a typo, cause in the pr I see
```cpp
#define DATE_LUT_MIN_YEAR 1925 /// 1925 since wast majority of timezones changed to 15-minute aligned offsets somewhere in 1924 or earlier.
#define DATE_LUT_MAX_YEAR 2283 /// Last supported year (complete)
```
@Enmk could you help, please?

Reply:
@KochetovNicolai Fixed that